src/transformation/builtins/array.ts

```diff
@@ -3,15 +3,13 @@ import * as lua from "../../LuaAST";
 import { TransformationContext } from "../context";
 import { UnsupportedProperty } from "../utils/errors";
 import { LuaLibFeature, transformLuaLibFunction } from "../utils/lualib";
-import { isExplicitArrayType } from "../utils/typescript";
 import { PropertyCallExpression, transformArguments } from "../visitors/call";
 
 export function transformArrayPrototypeCall(
     context: TransformationContext,
     node: PropertyCallExpression
 ): lua.CallExpression | undefined {
```

Comment:
```suggestion
): lua.CallExpression {
```

Reply:
That's not possible because the unsupported error is only thrown when isExplicitArrayType is true